pydantic/fields.py

```diff
@@ -20,9 +21,32 @@
 
 
 class FieldInfo(_repr.Representation):
-    """
-    Hold information about a field, FieldInfo is used however a field is defined, whether or not the `Field()`
-    function below is explicitly used.
+    """Hold information about a field.
```

Comment:
Thre are some function with docstring like
```
"""Docstring started in same line
```
and other functions
```
"""
Docstring started in new line
"""
I am not sure how is the diff when it will be rendered but I would suggest to make all of them in same format